recipes/nsls2-collection/meta.yaml

```
@@ -0,0 +1,121 @@
+{% set name = "nsls2-collection" %}
+{% set year = "2021" %}
+{% set cycle = "2" %}
+{% set version = "0" %}
+{% set micro = "" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ year }}C{{ cycle }}.{{ version }}{{ micro }}
+
+build:
+  skip: true  # [py<37 or not linux]
+  number: 0
+
+requirements:
+  host:
+    - python
+  run:
+    - nsls2-analysis ==2021C2.0=*_0
+    - python
+    - bloptools
+    - bluesky-darkframes
+    - caproto
+    - happi
+    - nslsii >=0.1.3
+    - pexpect
```

@kbeeperez, let's add one more package from our LCLS friends (https://github.com/slaclab/pydm).
```suggestion
    - pexpect
    - pydm
```